db/crud_test.go

```diff
@@ -1,14 +1,14 @@
 package db
 
 import (
-	"log"
-	"testing"
-
 	"github.com/couchbase/sync_gateway/base"
 	"github.com/couchbase/sync_gateway/channels"
 	goassert "github.com/couchbaselabs/go.assert"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"log"
```

Comment:
Keep the whitespace between native imports to maintain formatting.

Reply:
This has been fixed in the latest revision